ui/app/controllers/jobs/index.js

```diff
@@ -202,7 +195,7 @@ export default class JobsIndexController extends Controller {
    * @param {Error} e
    */
   notifyFetchError(e) {
-    const firstError = e.errors[0];
+    const firstError = e.errors?.objectAt(0);
```

Comment:
I should've caught this in https://github.com/hashicorp/nomad/pull/23427, but handling it here. Defensive logic in case there's an errors-less error (driving through a tunnel, etc.)